content/blog/eng/from-zero-to-infinite-the-final-keyword.md

```diff
@@ -129,21 +131,21 @@ class SomeImmutableObject
 }
 ```
 
-Let's concentrate, can we modify this object after it has been created? …. Yes we can!
+Let's concentrate: can we modify this object after it has been created? …. Yes we can!
 ```php
-$one = new SomeImmutableObject('Pippo');
-echo $one->getValue(); //Pippo
-$one->__construct('Pluto'); 
-echo $one->getValue(); //Pluto
+$one = new SomeImmutableObject('Foo');
+echo $one->getValue(); //Foo
+$one->__construct('Bar'); 
+echo $one->getValue(); //Bar
 ```
 
-try it! **[break the immutable object] (https://3v4l.org/CiHPG)**
+try it! **[break the immutable object](https://3v4l.org/AWndR)**
 
-This problem is easy to solve, we must put a flag in the constructor and if it is true throw an exception. 
-_Another way to fix it, is to create a `named constuctor` and make the `__constructor` private_. 
-let's do it using the first solution!
+This problem is easy to solve, we must put a flag in the constructor and if it is true, throw an exception. 
```

Comment:
This problem is easy to solve, we must put a flag in the constructor and, if it is true, throw an exception. 

Forse questa frase mi era sfuggita. Domanda: è il constructor il soggetto del verbo "throw"?
Potrebbe essere così? 
This problem is easy to solve: we must put a flag in the constructor and, if it is true, it throws an exception?

Reply:
Giustissimo